src/graphics/texture.js

```diff
@@ -133,8 +134,16 @@ Object.assign(pc, function () {
             this._height = (options.height !== undefined) ? options.height : this._height;
 
             this._format = (options.format !== undefined) ? options.format : this._format;
-            this.rgbm = (options.rgbm !== undefined) ? options.rgbm : this.rgbm;
-            this.swizzleGGGR = (options.swizzleGGGR !== undefined) ? options.swizzleGGGR : this.swizzleGGGR;
+
+            if (options.type !== undefined) {
+                this.type = options.type;
+            } else if (options.rgbm !== undefined) {
+                this.type = !!options.rgbm ? pc.TEXTURETYPE_RGBM : pc.TEXTURETYPE_DEFAULT;
+            } else if (options.rgbe !== undefined) {
+                this.type = !!options.rgbe ? pc.TEXTURETYPE_RGBE : pc.TEXTURETYPE_DEFAULT;
+            } else if (options.swizzleGGGR !== undefined) {
+                this.type = !!options.swizzleGGGR ? pc.TEXTURETYPE_SWIZZLEGGGR : pc.TEXTURETYPE_DEFAULT;
```

Comment:
OK, so from this code, I can see that `rgbm`, `rgbe` and `swizzleGGGR` are there for backwards compatibility. So it seems kind of weird to _add_ `rgbe` to the API at this point. If anything, I would remove them from the docs and print a deprecation warning if we want people to migrate to use the `type` property.

Reply:
Do we remove the docs for the deprecated flags or tag them as @deprecated?

Reply:
I normally just remove them. I'm not sure the `deprecated` tag does anything to the output docs HTML.

Reply:
done